### PR TITLE
`std::mem::drop` is in the prelude

### PR DESCRIPTION
**Pull request by mgeisler**

### CODE REVIEW

**Review by rbehjati — APPROVED** [review]

**Comment by qwandor** [comment]
I think this change makes the distinction between `std::mem::drop` and `Drop::drop` less clear.

**Comment by rbehjati** [comment]
> I think this change makes the distinction between `std::mem::drop` and `Drop::drop` less clear.

I agree with that :) 

I approved the change because I thought maybe there is an established convention that did not know of for shortening `std::mem::drop` to `drop`.

**Comment by qwandor** [comment]
> > I think this change makes the distinction between `std::mem::drop` and `Drop::drop` less clear.
> 
> I agree with that :)
> 
> I approved the change because I thought maybe there is an established convention that did not know of for shortening `std::mem::drop` to `drop`.

In code, sure. In the explanatory text here, I think it's more clear to be explicit.

File: src/traits/drop.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ fn main() {
 Discussion points:
 
 * Why doesn't `Drop::drop` take `self`?
-    * Short-answer: If it did, `std::mem::drop` would be called at the end of
+    * Short-answer: If it did, `drop` would be called at the end of
```
Comment (rbehjati) [inline src/traits/drop.md:37]: Perhaps change "`drop`" to "`drop` (std::mem::drop)" to be clear.

**Comment by mgeisler** [comment]
> > > I think this change makes the distinction between `std::mem::drop` and `Drop::drop` less clear.
> > 
> > 
> > I agree with that :)
> > I approved the change because I thought maybe there is an established convention that did not know of for shortening `std::mem::drop` to `drop`.
> 
> In code, sure. In the explanatory text here, I think it's more clear to be explicit.

Good point, I had not thought of distinguishing between the code and the text.

**Comment by mgeisler** [comment]
Oh, my bad, it's already merged... Let me fix in a new PR.

**Pull request merged**